qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg

```diff
@@ -47,11 +47,15 @@
             install_path = 'C:\Program Files (x86)'
         install_cmd = 'msiexec /i WIN_UTILS:\winfsp-1.7.20172.msi /qn'
         check_installed_cmd = 'dir "%s" |findstr /I winfsp'
-        viofs_log_file = C:\\viofs_log.txt
-        viofs_sc_create_cmd = 'sc create VirtioFsSvc binpath="%s -d -1 -D ${viofs_log_file}" start=auto'
+        viofs_log_file = C:\viofs_log.txt
+        viofs_sc_create_cmd = 'sc create VirtioFsSvc binpath="%s" start=auto'
         viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
         viofs_sc_start_cmd = 'sc start VirtioFsSvc'
         viofs_sc_query_cmd = 'sc query VirtioFsSvc'
+        viofs_reg_set_cmd = viofs_debug_enable_cmd, viofs_log_enable_cmd
```

Comment:
Actually, `viofs_reg_set_cmd` does not includes the specific command. not like `viofs_debug_enable_cmd`, `viofs_log_enable_cmd`, and so on, it looks like a set of commands or the order of commands to be executed.
I think we should rename it in a better way, like `viofs_reg_set_cmds_order`:
`viofs_reg_set_cmds_order = "${viofs_debug_enable_cmd}, ${viofs_log_enable_cmd}"`

or we don't need it at all.
The calling place we could do it like this:
```
viofs_debug_enable_cmd = params.get("viofs_debug_enable_cmd") 
viofs_log_enable_cmd = params.get("viofs_log_enable_cmd"):
if viofs_debug_enable_cmd and viofs_log_enable_cmd:
    ...
       for cmd in (viofs_debug_enable_cmd, viofs_log_enable_cmd):
       error_context.context("Set %s " % cmd, test.log.info)
       s, o = session.cmd_status_output(cmd)
```

Reply:
Thanks, update.